pkg/installmanager/installmanager.go

```diff
@@ -760,16 +789,16 @@ func (m *InstallManager) gatherClusterLogs(cd *hivev1.ClusterDeployment) error {
 	return err
 }
 
-func (m *InstallManager) initSSHKey() (string, error) {
+func (m *InstallManager) initSSHKey(envVar string) (string, error) {
 	m.log.Debug("checking for SSH private key")
-	sshPrivKeyPath := os.Getenv("SSH_PRIV_KEY_PATH")
+	sshPrivKeyPath := os.Getenv(envVar)
 	if sshPrivKeyPath == "" {
-		m.log.Warn("cannot configure SSH agent as SSH_PRIV_KEY_PATH is unset or empty")
-		return "", fmt.Errorf("cannot configure SSH agent as SSH_PRIV_KEY_PATH is unset or empty")
+		m.log.Warnf("cannot configure SSH agent as %s is unset or empty", envVar)
```

Comment:
This ends up generating a warn log entry here followed immediately by an info log entry in the caller with the same text.